interfaces/builtin/openvswitch_test.go

```diff
@@ -83,8 +83,4 @@ func (s *OpenvSwitchInterfaceSuite) TestUsedSecuritySystems(c *C) {
 	snippet, err := s.iface.ConnectedPlugSnippet(s.plug, s.slot, interfaces.SecurityAppArmor)
 	c.Assert(err, IsNil)
 	c.Assert(snippet, Not(IsNil))
-	// connected plugs have a non-nil security snippet for seccomp
```

Comment:
That test was bogus, openvswitch doesn't declare any seccomp snippets. The old commonInterface would return []byte("") if seccomp snippet is not defined.